src/FiscalYears/FiscalYearForm/FiscalYearForm.js

```diff
@@ -50,8 +53,21 @@ const FiscalYearForm = ({
   const closeForm = useCallback(() => onCancel(), [onCancel]);
 
   const validateCode = useCallback(
-    (value) => validateFYCode(ky, initialValues.id, value),
-    [initialValues.id, ky],
+    (fieldValue) => {
+      const errorMessage = <FormattedMessage id="ui-finance.fiscalYear.code.isInUse" />;
+      const params = {
+        ky,
+        api: FISCAL_YEARS_API,
+        id: initialValues.id,
+        fieldValue,
+        errorMessage,
+        fieldName: 'code',
+      };
+
+      return validateDuplicateFieldValue(params);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
with that comment you'll add usage of some var later and it will not give you a warning that it's missing in dependencies.